src/__tests__/abstract.test.ts

```diff
@@ -95,7 +95,7 @@ describe("Execute: Handles execution of abstract types", () => {
 
     const result = graphql(schema, query);
 
-    expect(result).toEqual({
+    expect(result).toMatchObject({
```

Comment:
I have read the description which mentions having to change toEqual to MatchObject due to differences in the GraphQL error object.

How does it related here with no errors?

Reply:
It was looking weird while some of tests still have `toEqual`.

Reply:
I am ok with mixing since toEqual is a much stronger check than MatchObject.
I would rather only change the ones that were failing.